### DISCUSSION TITLE
Relative time selector YESTERDAY in SOC

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#5883] [Ideas] [answered]
Hello!

My morning habbit is to take a look at all the alerts from yesterday and it would be AWESOME to have a YESTERDAY preset for the time span in SOC (Alert und Hunt view). No more fiddling around with the absolute time settings!

Cheers, Ben

**Comment by dougburks** [comment, marked as answer]
We might consider a YESTERDAY option at some point in the future but, in the meantime, have you considered setting it to Last 24 Hours? This is my morning habit.

**Reply by ben-sec** [reply]
Well, the last 24 hours is too relative and you probably miss something. I would get 100% coverage if I would do my review every day at 8:00 am, but there are days where I do my review later (e.g. 10:00 am) and then I would have a gap of 2 hours. Thanks for considering it anyway!

**Reply by dougburks** [reply]
To be clear, I use a different time span for Alerts and Hunt. For Alerts, I set a larger time span like 7 days or more and work that queue down to 0 alerts every day. For Hunt, I set a time span of last 24 hours and I am hunting at roughly the same time every day.

**Reply by dougburks** [reply]
Look for a YESTERDAY preset in the upcoming 2.3.90 release!
https://github.com/Security-Onion-Solutions/securityonion/issues/5953

**Reply by ben-sec** [reply]
You guys are awesome! Thanks a lot!
